NobelLaureateDetails/Pages/Index.cshtml.cs

```diff
@@ -16,7 +16,8 @@ public JsonResult OnGet()
 
             using (WebClient webClient = new WebClient())
             {
-                string PhysicsNinetyEight = webClient.DownloadString("http://api.nobelprize.org/v1/prize.json?category=physics&year=1998");
+                //string PhysicsNinetyEight = webClient.DownloadString("http://api.nobelprize.org/v1/prize.json?category=physics&year=1998");
```

Comment:
Remove commented out code.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.